### ISSUE TITLE
IpHelper::inRange IPv6 match error

### ISSUE THREAD

**Issue by mucsitamas** [yiisoft/yii2#17508]
### What steps will reproduce the problem?

I'm trying to use the inbuilt `IpHelper` class for checking IP addresses are in the given range or not. My range is `2620:0:2d0:200::7/32` and the IP I want to check is `2620:0:0:0:0:0:0:0`.

### What is the expected result?

The `2620:0:0:0:0:0:0:0` is the start range of the `2620:0:2d0:200::7/32` so it should contain it, the result should be `true`.

### What do you get instead?

```php
var_dump(\yii\helpers\IpHelper::inRange('2620:0:0:0:0:0:0:0', '2620:0:2d0:200::7/32')); // bool(false)
```

### Additional info

Checked Symfony and Java version, both gives true result for this range and address.

```php
var_dump(\Symfony\Component\HttpFoundation\IpUtils::checkIp('2620:0:0:0:0:0:0:0', '2620:0:2d0:200::7/32')); // bool(true)
```
For java: https://seancfoley.github.io/IPAddress/
```java
import inet.ipaddr.IPAddressString;

public class Main {
    public static void main(String [] args) {
        final boolean result = matches("2620:0:0:0:0:0:0:0","2620:0:2d0:200::7/32");
        System.out.println(String.format("Result: %s", result ? "match" : "not match")); // "match"
    }

    private static boolean matches(String ip, String subnet) {
        IPAddressString one = new IPAddressString(subnet);
        IPAddressString two = new IPAddressString(ip);
        return one.getAddress().contains(two.getAddress());
    }
}
```

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.15.1
| PHP version      | 7.2.19
| Operating system | Ubuntu 18.04.3 LTS

**Comment by Thoulah** [comment]
According to [CIDR to IPv6 Conversion](https://www.ipaddressguide.com/ipv6-cidr), the first IP of 2620:0:2d0:200::7/32 is 2620:0:2d0:200::7, so I would say 2620::0 is indeed not in this range.

**Comment by samdark** [comment]
@Thoulah I think [that tool is wrong](https://www.vultr.com/resources/subnet-calculator-ipv6/?prefix_length=64&display=long&ipv6_address=2620%3A0%3A2d0%3A200%3A%3A7%2F32).

**Comment by Thoulah** [comment]
Fair enough! First hit in Google failed me.

**Comment by kamarton** [comment]
fixed, please merge.

**Comment by SilverFire** [comment]
Merged in #17511 

@mucsitamas thanks for the report, @kamarton thank you for a fix!
